sources/PseudoTerminal.m

```diff
@@ -592,6 +593,32 @@ - (void)finishInitializationWithSmartLayout:(BOOL)smartLayout
 
     [self updateTabBarStyle];
     self.window.delegate = self;
+    
+    if(windowType == WINDOW_TYPE_NO_TITLE_BAR_ROUNDED) {
+        // Hide the titlebar
+        self.window.titlebarAppearsTransparent = true;
+        
+        // Hide the titlebar
+        self.window.titlebarAppearsTransparent = true;
+        
+        // Hide the title
+        self.window.titleVisibility = NSWindowTitleHidden;
+        
+        // Hide the close button
+        NSButton* closeButton = [[self window] standardWindowButton:NSWindowCloseButton];;
```

Comment:
Please put space before the *
There are two semicolons

Same on lines 612 and 616